### DISCUSSION TITLE
brew dependabot trigger

### DISCUSSION THREAD

**Discussion by solaris7x** [Homebrew/discussions#1869] [Everyday usage] [answered]
Hello, 
I'm using brew to install some packages (screen,vim,htop) in a docker container (ubuntu-20 base image) and I would like to trigger a new build for whenever a new version is available for the package. I was hoping this could be done by github dependabot but it needs a package-vendor (eg. npm). Has anyone tried this before ? Any help much appreciated

**Comment by SMillerDev** [comment, marked as answer]
Why not just trigger the build every day if the version in the brew api differs from your version?

**Reply by solaris7x** [reply]
That works too , is there a sample or docs related to this ?

**Reply by SMillerDev** [reply]
https://formulae.brew.sh/docs/api/
